tensorrt_llm/serve/openai_server.py

```
@@ -631,6 +648,124 @@ def _check_health(self) -> bool:
+        generated. The transfer runs in the background -- the response does not
+        wait for it.
+
+        Returns 202 with {status, request_hash}, where status is one of
+        `accepted`, `deduped` (an identical prompt was seen within the dedup
+        TTL), or `skipped_busy` (the in-flight transfer cap was reached).
+        Returns 400 if `source` or the prompt is missing, 403 if `source` is
+        not in the configured allowlist.
+        """
+        try:
+            body = await request.json()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)